.github/workflows/pre-release-tests.yaml

```diff
@@ -0,0 +1,55 @@
+name: Pre Release tests
+on:
+  workflow_dispatch:
```

Comment:
Is this intentionally the only thing that triggers this workflow? Can we automate them somehow... maybe once a week on the weekend?